docs/tutorial/kymotracking.rst

```diff
@@ -110,6 +116,55 @@ Fortunately, the signal to noise level in this kymograph is quite good. In pract
 one will have to resort to some fine tuning of the algorithm parameters over different regions of the kymograph to get
 an acceptable result.
 
+Gaussian Maximum Likelihood Estimation
```

Comment:
I find Gaussian Maximum Likelihood Estimation confusing. I would omit the word Gaussian. Since it makes me think you are assuming a Gaussian noise model which is exactly not what you're doing :D